lib/client/sse_client.dart

```diff
@@ -79,8 +86,8 @@ class SseClient extends StreamChannelMixin<String> {
     var encoded = jsonEncode(message);
     try {
       await _client.post(_serverUrl, body: encoded);
-    } catch (_) {
-      // Ignore any error
+    } catch (e) {
+      _logger.warning('Unable to encode outgoing message: $e');
```

Comment:
Note that this won't actually log anything anywhere by default... not necessarily an issue in itself but it might be worth documenting somewhere that you have to set up a log handler to see logs?

Reply:
Added a corresponding doc comment.